src/assisted_test_infra/test_infra/tools/terraform_utils.py

```diff
@@ -11,12 +11,25 @@
 from service_client import log
 
 
+class _Terraform(Terraform):
+    """python_terraform.Terraform always set the force flag (even to false) causing
+    destroy failures on some cases. This class is overrides the destroy method and set the
+    force flag only if it's set to true"""
+    def destroy(self, dir_or_plan=None, force=False, **kwargs):
```

Comment:
```suggestion

    def destroy(self, dir_or_plan=None, force=False, **kwargs):
```
`black` wants a blank line here

Reply:
Yeah, sorry, missed that :|